pages/service.md

```diff
@@ -0,0 +1,7 @@
+# Service Area
+
+You can learn about the broad range of services that [CivicActions our central website](https://civicactions.com/approach). CivicActions has deep experience with open source software and agile development - both lend themselves to great accessibility practices. 
```

Comment:
@mgifford I think the first sentence needs to be tweaked, maybe just change 'that' to 'at'?